app/main.py

```diff
@@ -1,3 +1,79 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, distance: Distance) -> None:
+        self.km = distance
```

Comment:
Inside of your `__init__` method you take the arguments that are needed to initialise your object's class, i.e. there must be `km` of type int | float